index.html

```diff
@@ -190,9 +191,12 @@ <h2>Contact me</h2>
             <input
               type="email"
               id="email"
+              title="The email must be in all lowercase letters."
+              pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$"
               required
               placeholder="loremipsum@gmail.com"
             />
+            <p id="error"></p>
           </li>
           <li>
             <textarea
```

Comment:
- [ ] nice work so far 👍🏿  but, as the previous code reviewer has pointed out your applications seems not to still be validating the form, please do well to fix it. Here is an example on how to validate your form
 ```
const email = document.querySelector('#email');
const form = document.querySelector('.contact-us');
const error = email.nextElementSibling;

const inputPatern = /^[a-z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-z0-9-]+(?:\.[a-z0-9-]+)*$/;

form.addEventListener('submit', (event) => {
    const valid = inputPatern.test(email.value);
    if (!valid) {
        error.style.height = "50px";
        error.style.padding = "10px";
        error.innerHTML = '<i class="fa-solid fa-triangle-exclamation"></i>  Please make sure you enter your email in lowercase!';
        event.preventDefault();
    } 
    else {
        error.innerHTML = '';
    }
});
```

Reply:
Please check the validations.js, thank you.